content/docs/sidebar.json

```diff
@@ -33,6 +33,10 @@
     "label": "Self-hosted Runners",
     "slug": "self-hosted-runners"
   },
+  {
+    "label": "Install as a Package",
+    "slug": "install"
+  },
   {
     "label": "Command Reference",
     "slug": "ref",
```

Comment:
Moved this entry one place up. Not a strong opinion, but I think it makes more sense as it's similar to the other lose pages (guides, I guess?).